README.md

```diff
@@ -1 +1,50 @@
-# UnitA Project
+# Unit A landing page
+
+_This project is a landing page for a branch of __APE UNIT__ company which is an international tech company based in Germany_
```

Comment:
Just saying...
this should state the actual description on the website
`A joint-force between African & European developers & organizations working in blockchain technology`